src/api/functions.jl

```diff
@@ -2440,3 +2517,80 @@ function h5z_register(filter_class)
     return nothing
 end
 
+"""
+    h5fd_core_init() -> hid_t
+
+See `libhdf5` documentation for [`H5FD_core_init`](https://portal.hdfgroup.org/display/HDF5/H5FD_CORE_INIT).
```

Comment:
Doesn't look like any of these hd5f_* methods are defined in the docs.

Reply:
They're not: they're API is exposed via macros, e.g.:
https://github.com/HDFGroup/hdf5/blob/1d680fe04c545d601678d876ad22dea7bfc31edd/src/H5FDcore.h#L23
the only place I see them in the docs is https://portal.hdfgroup.org/display/HDF5/H5P_GET_DRIVER